src/textual/screen.py

```diff
@@ -184,6 +186,11 @@ class Screen(Generic[ScreenResultType], Widget):
 
     Should be a set of [`command.Provider`][textual.command.Provider] classes.
     """
+    ALLOW_IN_MAXIMIZED_VIEW: ClassVar[str] = ".-textual-system,Footer"
```

Comment:
Maybe?
```suggestion
    ALLOW_IN_MAXIMIZED_VIEW: ClassVar[str] = ".-textual-system,Header,Footer"
```

Reply:
The Header seemed less important. But the Footer contains stuff that may be required for the maximized widget.

I think its best to leave that to the dev.